react-native-gutenberg-bridge/android/src/main/java/org/wordpress/mobile/ReactNativeGutenbergBridge/RNReactNativeGutenbergBridgePackage.java

```diff
@@ -12,19 +12,23 @@
 public class RNReactNativeGutenbergBridgePackage implements ReactPackage {
     private GutenbergBridgeJS2Parent mGutenbergBridgeJS2Parent;
     private RNReactNativeGutenbergBridgeModule mRNReactNativeGutenbergBridgeModule;
+    private boolean mIsDarkMode;
```

Comment:
Would be nice if we could marm `mIsDarkMode` and `mGutenbergBridgeJS2Parent` as `final`. Would make it clear that `mRNReactNativeGutenbergBridgeModule` is the only thing that can/should change. I do not feel strongly about this change though, feel free to leave as-is if you prefer.

Reply:
Thanks for the good advice.
Will update them!